Client/Frontend/Browser/HomePanel/favorites/FavoritesDataSource.swift

```diff
@@ -133,6 +133,7 @@ extension FavoritesDataSource: NSFetchedResultsControllerDelegate {
             // otherwise it crashes with inconsistency exception.
             if let indexPath = newIndexPath, indexPath.row < columnsPerRow {
                 collectionView?.insertItems(at: [indexPath])
+                collectionView?.reloadData()
```

Comment:
Kind of related to http://www.openradar.me/15262692
Without this, on iOS 12 `NSInternalInconsistencyException` error is thrown when creating favorites